pkg/controller/migcluster/validation.go

```diff
@@ -216,6 +226,50 @@ func (r ReconcileMigCluster) testConnection(cluster *migapi.MigCluster) error {
 	return nil
 }
 
+// Validate the Exposed registry route
+func (r ReconcileMigCluster) validateRegistryRoute(cluster *migapi.MigCluster) error {
+	if cluster.Spec.IsHostCluster {
```

Comment:
I suppose the registry route for the host cluster is not user-provided based on this?

Reply:
The UI provides no way for specifying it, and for most purposes it's not needed there since we just use the internal registry path. There is one scenario where the host cluster does require it, though: If the controller is run via `make run` and the host cluster is either a source or destination cluster, then an ExposedRegistryPath must be provided, since the controller running locally won't be able to access cluster-internal service ports. This will require the user to manually set this parameter on the MigCluster (since the UI doesn't allow editing the host migcluster). This isn't a supported scenario, since we don't support customers running `make run`, but it's relevant for developers (who can `oc edit` the cluster easily enough). Back to the topic of this PR, though, the above is fine if we don't need to run this validation for this non-standard developer-only scenario. Alternatively, we could replace `if cluster.Spec.IsHostCluster` with `if cluster.Spec.ExposedRegistryPath != "" `

Reply:
Just noticing that below here we don't do anything if the exposed registry path is empty, so we may want to just eliminate the IsHostCluster test.